core/src/io/anuke/mindustry/game/Stats.java

```diff
@@ -22,6 +22,14 @@
     public int buildingsDeconstructed;
     /** Friendly buildings destroyed. */
     public int buildingsDestroyed;
+    /** Players killed */
+    public int playersKilled;
+    /** Cores destoryed */
+    public int coresDestroyed;
+    /** Resources spent */
+    public int resourcesSpent;
+    /** Place */
```

Comment:
Looking over these, they don't seem too useful-- instead of keeping track of total players killed, it should be on a ally-enemy bases, e.g. "Team player deaths" and "Enemy player deaths". That gives a much better overview of how the fight went.

I also don't see how "cores destroyed" is needed. Most maps have less than 3 cores, so it's not particularly exciting to find out the total number of destructions.

Also, these are... not very helpful comments. If you're just going to re-state the name of the variable, it's better to have no comments at all. I would suggest:

- "*Enemy/Friendly* players killed"
- "*Enemy/Friendly* cores destroyed"
- "Total items spent on buildings"
- What is "place"? Is it the ranking of the team relative to other teams? If so, you should indicate that.